example/test_problem/Hydro/Cosmic_Ray_Diffusion/generate_make.sh

```diff
@@ -0,0 +1,6 @@
+# This script should run in the same directory as configure.py
+
+PYTHON=python
+
+${PYTHON} configure.py --machine=eureka_intel --model=HYDRO --flu_scheme=MHM_RP --flux=HLLD --mhd=true \
```

Comment:
`--hdf5=true`

Reply:
There will be error in `Init/Init_ByRestart_HDF5.cpp` and it cannot be compiled.

Reply:
Updated! The bug is also fixed.